source/dlangbot/app.d

```diff
@@ -193,7 +193,12 @@ void handlePR(string action, PullRequest pr)
     auto refs = getIssueRefs(commits);
 
     auto descs = getDescriptions(refs);
-    updateGithubComment(action, refs, descs, pr.commentsURL);
+    auto comment = pr.getBotComment;
+
+    pr.updateGithubComment(comment, action, refs, descs);
+
+    if (refs.length > 0 && comment.body_.length == 0)
+        pr.addLabels(["Bug fix"]);
```

Comment:
You should check that the referenced tickets are actually fixed, `refs.any!(r => r.fixed)` should work.
https://github.com/dlang-bots/dlang-bot/blob/47e16c0a921e69e2f65abd1fb03015ea78975666/source/dlangbot/bugzilla.d#L37